app/connectors/CustomsDeclareExportsConnector.scala

```diff
@@ -28,16 +28,23 @@ import scala.concurrent.{ExecutionContext, Future}
 @Singleton
 class CustomsDeclareExportsConnector @Inject()(appConfig: AppConfig, httpClient: HttpClient) {
 
+  private[connectors] def get(url: String)(implicit hc: HeaderCarrier, ec: ExecutionContext) =
+    httpClient.GET(url, Seq())
+
+  private[connectors] def postSubmission(
+    body: Submission
+  )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[CustomsDeclareExportsResponse] =
+    httpClient.POST[Submission, CustomsDeclareExportsResponse](
+      s"${appConfig.customsDeclareExports}${appConfig.saveSubmissionResponse}", body, Seq()
+    )
+
   def saveSubmissionResponse(body: Submission)
                             (implicit hc: HeaderCarrier, ec: ExecutionContext): Future[CustomsDeclareExportsResponse] =
-    post(body).map{ response =>
+    postSubmission(body).map { response =>
       Logger.debug(s"CUSTOMS_DECLARE_EXPORSTS response is --> ${response.toString}")
       response
     }
 
-  private[connectors] def post(body: Submission)
-                              (implicit hc: HeaderCarrier, ec: ExecutionContext): Future[CustomsDeclareExportsResponse] =
-    httpClient.POST[Submission, CustomsDeclareExportsResponse](
-      s"${appConfig.customsDeclareExports}${appConfig.saveSubmissionResponse}", body, Seq()
-    )
+  def fetchNotifications(eori: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Notifications] =
+    httpClient.GET[Notifications](s"")
```

Comment:
I don't know what endpoint will be in our backend app, so it will be added when endpoint will works